ffmpeg_normalize/_streams.py

```diff
@@ -481,10 +481,26 @@ def get_second_pass_opts_ebu(self) -> str:
                 "Specify -ar/--sample-rate to override it."
             )
 
+        target_level = self.ffmpeg_normalize.target_level
+        if self.ffmpeg_normalize.safe_ebu > 0:
+            safe_target = (
+                self.loudness_statistics["ebu_pass1"]["input_i"]
+                - self.loudness_statistics["ebu_pass1"]["input_tp"]
+                + self.ffmpeg_normalize.true_peak
+                - self.ffmpeg_normalize.safe_ebu
+            )
+            if safe_target < self.ffmpeg_normalize.target_level:
+                _logger.warning(
+                    "Using loudness target %s because --safe-ebu=%s.",
```

Comment:
The rest of the code base uses `{}` formatting rather than printf-style formatting, but no need to change it.